tools/synthgen/internal/synthgen/negative_test.go

```
@@ -0,0 +1,374 @@
+		switch s {
+		case "a":
+			u = 0xaf63dc4c8601ec8c
+		default:
+			u = 0xaf63df4c8601f1a5
+		}
+		return int64(u)
+	}
+	if NegSeed("a") != fnvOf("a") || NegSeed("b") != fnvOf("b") {
+		t.Fatalf("NegSeed FNV-1a 向量锚定失效: a=%#x b=%#x", NegSeed("a"), NegSeed("b"))
+	}
+	if NegSeed("T4-G0-01") != NegSeed("T4-G0-01") {
```

golangci-lint (2.12.2)

[error] SA4000: identical expressions on the left and right side of the '!=' operator

(staticcheck)